src/emails/getEmailL10n.ts

```diff
@@ -0,0 +1,20 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { SubscriberRow } from "knex/types/tables";
+import { acceptedLanguages, negotiateLanguages } from "@fluent/langneg";
+import { ExtendedReactLocalization } from "../app/hooks/l10n";
+import { getSpecificL10nSync } from "../app/functions/server/mockL10n";
+
+export function getEmailL10n(
+  subscriber: SubscriberRow,
```

Comment:
Could we use a more specific type, e.g. using `Pick`? `SubscriberRow` is fairly broad and contains data we wouldn't want to appear in an email.